Makefile

```diff
@@ -44,6 +44,15 @@ rundev: builddev
 stop:
 		@docker-compose down
 
+runtest: rundev
+		@sleep 1
+		@docker-compose exec -T web ./vendor/phpunit/phpunit/phpunit --bootstrap ./bootstrap.php --configuration ./phpunit.xml
+
+runcoverage: rundev
```

Comment:
Because it is not specified here.

Reply:
This isn't run in GitHub actions. It behaved slightly different even though it should not

Reply:
(referring to #1297 )